voctocore/lib/streamblanker.py

```diff
@@ -16,28 +16,30 @@ def __init__(self):
         self.log.info('Configuring StreamBlanker video %u Sources',
                       len(self.names))
 
+        self.volume = Config.getfloat('stream-blanker', 'volume', fallback=1.0)
```

Comment:
yes, correct. I removed the fallback.
I'm more in to having a file, because
 - you only have on place to define a default (and not potentially multiple defaults all over the code)
 - the default-config serves as a living documentation which is usually not out of date and can directly be copied and modified